### PR TITLE
Update example to work with SDK 0.44.x

### LINKED ISSUE #18: Update example work with SDK 0.44.x
Hello!
Since updating the sdk from cosmos 0.44.x, your example no longer works, so I updated it.
I added the proto and stargate lib of cosmJs and adapted the `connectWithSigner()` function.

If that can help! I can do a clean pull request with atom data.

Check this commit: https://github.com/atmoner/keplr-example/commit/7ba6888d040a373d8f68d0ba032ef7a4135af814

Best regards, 
atmon3r | BitCanna

### PR DESCRIPTION
**Pull request by daniel-farina**
- Updates example code to work with stargate (Credit goes to https://github.com/atmoner/keplr-example/commit/7ba6888d040a373d8f68d0ba032ef7a4135af814)
- Updated package.json to include missing packages
- Update layout
- Add readme

I made this particular example work with OSMO with the open endpoints that are available with https://rpc-osmosis.blockapsis.com, but this can be changed to work with any chain. 

Closes #18

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,4 +1,14 @@
-const { SigningCosmosClient } = require("@cosmjs/launchpad");
+const {
+    SigningCosmosClient
+} = require("@cosmjs/launchpad");
```
Comment (delivan) [inline src/main.js:3]: It would be good to change`import { SigningCosmosClient } = from '@cosmjs/launchpad'`

**Review by delivan — CHANGES_REQUESTED** [review]

**Comment by alighasemzadeh** [comment]
@daniel-farina 
Good pull request, just  alert for successful transactions not disapplied after uncomment.
Please Give Me Working Example:
https://github.com/daniel-farina/keplr-example/blob/ebe342bbc96fb376b880c43d045bf64081bfcf2f/src/main.js#L195

File: src/main.js (hunk 1 above)
Comment (daniel-farina) [reply src/main.js]: Fixed, thanks

**Comment by daniel-farina** [comment]
> @daniel-farina Good pull request, just alert for successful transactions not disapplied after uncomment. Which related to: https://github.com/daniel-farina/keplr-example/blob/ebe342bbc96fb376b880c43d045bf64081bfcf2f/src/main.js#L195
> 
> I think we have to remove: https://github.com/daniel-farina/keplr-example/blob/ebe342bbc96fb376b880c43d045bf64081bfcf2f/src/main.js#L189
> 
> After removing this live everything's work well. For Successful Transactions `alert("Succeed to send tx:" + result.transactionHash);` is good message.

Thank you, made the code changes suggested.

**Comment by alighasemzadeh** [comment]
Please merge my pull request to your repo after that try new pull request.

**Comment by daniel-farina** [comment]
> @daniel-farina Please merge my pull request to your repo after that try new pull request.

I see it now, merged & thanks @alighasemzadeh

**Pull request merged**